zhaquirks/tuya/__init__.py

```diff
@@ -354,47 +354,36 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
 
-class TuyaSmartRemoteOnOffCluster(EventableCluster):
-    """TuyaSmartRemoteOnOffCluster: this cluster manipulates messages from the remote control and converts them to command_ids."""
+class TuyaSmartRemoteOnOffCluster(OnOff, EventableCluster):
+    """TuyaSmartRemoteOnOffCluster: fire events corresponding to press type."""
 
-    cluster_id = 0x0006
+    press_type = {
+        0x00: SHORT_PRESS,
+        0x01: DOUBLE_PRESS,
+        0x03: LONG_PRESS,
```

Comment:
This was 0x02 previously. Is this change intentional?

Reply:
Good catch. I'll fix it.

Reply:
fixed in https://github.com/zigpy/zha-device-handlers/pull/673/commits/14ac22d66f6033e0fc9340984676edc2c645b057